Resources/Private/Partials/Backend/Application/Field/SummaryListFiles.html

```diff
@@ -9,7 +9,7 @@
 			<f:for each="{application.files}" as="file">
 				<li class="list-group-item">
 					<a href="{file.backendUrl}" target="_blank">
-						<f:image src="{file.backendUrl}" maxHeight="100" style="border: 1px solid #DDD;" /> {file.originalResource.name}
+						<f:image src="{file.originalResource.uid}" maxHeight="100" style="border: 1px solid #DDD;" treatIdAsReference="TRUE"/> {file.originalResource.name}
```

Comment:
Please use the `image` attribute instead and remove the `treatIdAsReference` attribute.